### DISCUSSION TITLE
DriverStoreExplorer - Chocolatey Package?

### DISCUSSION THREAD

**Discussion by rorar** [lostindark/DriverStoreExplorer#165] [Ideas]
Hi there,

it would be great to have DriverStoreExplorer in the package manager Chocolatey :-) 
@lostindark would you consider implementing it?

All the best

**Comment by lostindark** [comment]
What is the use case?

**Reply by katastrophe92** [reply]
Hi! I also love package managers to use them with [WingetUI](https://github.com/marticliment/WingetUI). Easier to install/restore/automate installations, also from a restore-file.

**Comment by lostindark** [comment]
No plan to support Chocolatey.
It looks like bettydreemurr maintains a winget package for DriverStoreExplorer.
You can install via:
`winget install lostindark.DriverStoreExplorer`
